src/router/router.js

```
@@ -0,0 +1,28 @@
+  "/error": { layout: AuthLayout, component: ErrorPage },
+};
+
+// 각 라우트에 해당하는 컴포넌트 렌더링 함수
+export const navigateTo = (path) => {
+  const isInvalidPath =
+    !path || !path.startsWith("/") || path.startsWith("//") || !routes[path];
+  if (isInvalidPath) {
+    path = "/error";
+  }
+
+  history.pushState(null, null, path);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)